AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -420,6 +420,16 @@ - (void)channel:(id<MSChannelProtocol>)__unused channel
     return;
   }
 
+  // FIXME: Rejecting MSCommonSchemaLog in the buffer for now.
+  /*
+   * Don't buffer logs that are MSCommonSchemaLogs or contain one or more transmissiontargetTokens for now. The reason
+   * is that the crash buffer currently cannot handle MSCommonSchemaLogs. It needs significant work, e.g. for
+   * encrypting/decrypting the target tokens. Putting this in a separate paragraph for visibility.
+   */
+  if ([logObject isKindOfClass:[MSCommonSchemaLog class]] || log.transmissionTargetTokens) {
```

Comment:
You can also remove the `isKindOfClass:[MSCommonSchemaLog class]` test, testing existence of transmission token should be enough, AC only logs won't set it.